results/proto/pipelinerun.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package tekton.pipeline.v1;
+
+import "common.proto";
+import "taskrun.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "./proto";
+
+message PipelineRun {
+    string api_version = 1;
+	string kind = 2; // TODO: must always be TaskRun?
+
+	ObjectMeta metadata = 3;
+
+	PipelineRunSpec spec = 4;
+	PipelineRunStatus status = 5;
+}
+
+message PipelineRunSpec {
+    PipelineSpec task_spec = 5;
+	google.protobuf.Duration timeout = 7;
+}
+
+message PipelineSpec {
+    repeated PipelineDeclaredResource resources = 2;
+    repeated PipelineTask tasks = 3;
+    repeated PipelineResult results = 6;
+    repeated PipelineTask final_tasks = 7;
+}
+
+message PipelineTask {
+    string name = 1;
+    TaskRef task_ref = 2;
+    EmbeddedTask task_spec = 3;
+
+    google.protobuf.Duration timeout = 11;
+}
+
+message TaskRef{
+    string name = 1;
+    string kind = 2;
+    string api_version = 3;
+}
+
+message EmbeddedTask{
+    map<string, string> labels = 1;
+    map<string, string> annotations = 2;
+    TaskSpec task_spec = 3;
+}
+
+message PipelineResult {
+    string name = 1;
+    string description = 2;
+    string value = 3;
+}
+
+message PipelineDeclaredResource {}
+
+message PipelineRunStatus {
+    repeated Condition conditions = 1;
+    int64 observed_generation = 2;
+	map<string, string> annotations = 3;
+	google.protobuf.Timestamp start_time = 4;
+    google.protobuf.Timestamp completion_time = 5;
+    
+    // The type of this field in go struct is map[string] *PipelineRunTaskRunStatus
+    map<string, PipelineRunTaskRunStatus> taskruns = 6;
+    PipelineSpec pipeline_spec = 7;
+}
+
+message PipelineRunTaskRunStatus {
+    string pipeline_task_nam = 1;
```

Comment:
```suggestion
    string pipeline_task_name = 1;
```

We should probably add some sort of test to make sure this converts correctly. With TaskRuns there were a few cases where the Go struct name was not in the default format protobuf expected, causing issues when converting.